functions/_nvm_auto_invoke.fish

```diff
@@ -1,5 +1,8 @@
 function _nvm_auto_invoke \
     --on-variable PWD \
     --description 'Use Node.js version specified by project automatically'
-    nvm use 2>/tmp/_nvm_auto_invoke.log
+
+    if set --query auto_invoke_nvm; and test "$auto_invoke_nvm" = true
```

Comment:
I'd rather have the flag prevent `_nvm_auto_invoke` from being defined at all, rather than define the --on-variable PWD hooks and incur this check every time.

Reply:
Doing something like below break the autoloading mechanism:

```
if set --query nvm_use_on_pwd_change; and test "$nvm_use_on_pwd_change" = true
    function _nvm_use_on_pwd_change \
        --on-variable PWD \
…
    end
end
```

Should I hook `_nvm_use_on_pwd_chang` on both the flag variable (`$nvm_use_on_pwd_change`) AND `$PWD`?